packages/core/src/command/consolidated/consolidated-get.ts

```diff
@@ -0,0 +1,47 @@
+import { Bundle, Resource } from "@medplum/fhirtypes";
+import { createConsolidatedDataFilePath } from "../../domain/consolidated/filename";
+import { executeWithRetriesS3, returnUndefinedOn404, S3Utils } from "../../external/aws/s3";
+import { Config } from "../../util/config";
+
+const defaultS3RetriesConfig = {
+  maxAttempts: 3,
+  initialDelay: 500,
+};
+
+export type Consolidated = {
+  bundle: Bundle<Resource> | undefined;
+  fileLocation: string;
+  fileName: string;
+};
+
+export async function getConsolidated({
+  cxId,
+  patientId,
+  fileLocation = Config.getMedicalDocumentsBucketName(),
+  region = Config.getAWSRegion(),
+  log = console.log,
+}: {
+  cxId: string;
+  patientId: string;
+  fileLocation?: string;
+  region?: string;
+  log?: typeof console.log;
+}): Promise<Consolidated> {
+  const s3Utils = new S3Utils(region);
+  const fileName = createConsolidatedDataFilePath(cxId, patientId);
+  const consolidatedDataRaw = await executeWithRetriesS3<string | undefined>(
+    async () => returnUndefinedOn404(() => s3Utils.getFileContentsAsString(fileLocation, fileName)),
```

Comment:
do we want / need to wrap aws sdk i/o calls in try catch blocks

Reply:
Why? Not everything needs try/catch, it will bubble up. What do you think we should catch/do here if there's an error?

Reply:
Yah I guess it will bubble up all the way to `sqs-to-consolidated.ts`
All good then